server/tserver/src/main/java/org/apache/accumulo/tserver/compactions/CompactionService.java

```diff
@@ -363,11 +374,11 @@ private void planCompaction(CompactionKind kind, Compactable compactable,
 
       if (!jobs.isEmpty()) {
         log.trace("Submitted compaction plan {} id:{} files:{} plan:{}", compactable.getExtent(),
-            myId, files, plan);
+            myId, plan.getCandidates(), plan);
```

Comment:
With changes in other comment could revert to the following.  Also this changes is logging less information than it used to. Used to log the Compactable.Files obj now it logging a subset of that info.

```suggestion
            myId, files, plan);
```

Reply:
I reverted the changes to CompactionPlan but here I changed the method to use the `Compactable.Files` object as opposed to the Optional. I think this cleans it up a bit and should print better info than printing the Optional object.

Reply:
I also noticed printing the `Optional.empty` after we check if its empty just clutters the log message, so I dropped it:
<pre>
TRACE: Compactable returned no files 4;row_0003250000;row_0003000000 USER Optional.empty
TRACE: Compactable returned no files 4;row_0003250000;row_0003000000 CHOP Optional.empty
TRACE: Compactable returned no files 4;row_0003500000;row_0003250000 SYSTEM Optional.empty
TRACE: Compactable returned no files 4;row_0003500000;row_0003250000 SELECTOR Optional.empty
</pre>